Clickable not working anymore

### Description
There's an error when building via clickable.
I think it's related to this #473 

```
--   No package 'libapparmor' found
CMake Error at /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:367 (message):
  A required package was not found
Call Stack (most recent call first):
  /usr/share/cmake-3.5/Modules/FindPkgConfig.cmake:532 (_pkg_check_modules_internal)
  src/app/CMakeLists.txt:12 (pkg_check_modules)


-- Configuring incomplete, errors occurred!

```

--- a/clickable.json
+++ b/clickable.json
@@ -4,5 +4,6 @@
   "build_args": "-DCLICK_MODE=ON",
   "dependencies_target": [
     "qtwebengine5-dev"
+    ,"libapparmor-dev"
   ]
 }

